composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
the curl stuff has 100% been broken before, so unless we've got a great way to mock it then i think this is fine. We do integration tests all over the place, and they're far superior to unit tests especially if you can make them fast.